public/amls.css

```diff
@@ -1353,6 +1353,11 @@ table.notifications thead td {
   text-decoration: underline;
 }
 
+.message-inbox__list
```

Comment:
Is this line needed? It should have an underline until it has focus (I'm flying blind here! Might be worth my looking over your shoulder tomorrow morning)

Reply:
it works as you wanted I believe, but we can have a look together when you're in